design-library/src/components/BccTabs/BccTabs.vue

```diff
@@ -18,6 +18,7 @@ import type { Component } from "vue";
 type Props = {
   tabs: BccTabsGroup;
   size?: keyof typeof sizes;
+  fill?: boolean;
```

Comment:
Probably should have a default of false? Otherwise it's undefined which is also falsy but makes it a little clearer in the documentation, we do it for other boolean values as well.